src/main/java/tars/Tars.java

```diff
@@ -0,0 +1,99 @@
+package tars;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * This class is Chatbot application which handles Task Management, mainly ToDos, Deadline and Event tasks
+ * Users can add the 3 different type of tasks and carry out other functions like mark, unmark and delete
+ * Users can also view all the tasks added in a LIST
+ * This application will run till "bye" command given
+ *
+ * @author csk
+ * @version 1
+ */
+public class Tars {
+    private TaskList taskList;
+    private Storage storage;
+    private Ui ui;
+    private Parser parserHelp;
+
+    /**
+     * Constructs the Tars object, taking in the filePath for storage and initialising Ui and Parsers objects
+     *
+     * @param filePath
+     * @throws IOException
+     */
+    public Tars(String filePath) throws IOException {
+        this.ui = new Ui();
+        storage = new Storage(filePath);
+        this.parserHelp = new Parser();
+        try {
+            taskList = new TaskList(storage.readFile());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Runs TARS application with using the different objects(eg: Ui, storage, taskList)
+     * Handles scanning the input of user and execute relevant command based on input
+     *
+     * @throws IOException
+     */
+    /*public void run() throws IOException {
```

Comment:
do remove this block comment if you decide to scrap this method to ensure your code looks clean (it will be here, on github if you want to recover it)